dev_guide/projects.adoc

```diff
@@ -26,8 +26,7 @@ Constraints:: quota for the project, limit pods in a project to a pool of nodes
 To create a new project:
 
 ****
-`$ osc new-project _<project_name>_ --description="_<description>_"
---display-name="_<display_name>_" --node-selector="_<label>_"`
+$ osc new-project _<project_name>_ --description="_<description>_" --display-name="_<display_name>_" --node-selector="_<label>_"`
```

Comment:
This looks like it's a mix of the old and new guidelines for command syntax. Should use `----` and not use backticks (there's one on the end).